README.md

```diff
@@ -529,8 +529,9 @@ func main() {
 	if err != nil {
 		log.Fatalf("create payment error: %v", err)
 	}
-
-	_, err = pay.GenerateTx(client)
+    
+    systemScripts, _ := utils.NewSystemScripts(client)
```

Comment:
Not align?

Reply:
The format is normal in the code, but the format is indeed messed up in the preview, I will adjust it, thanks.

Reply:
updated https://github.com/nervosnetwork/ckb-sdk-go/pull/16/commits/a2152bebcc2f6dceb88d714e6776050f08fbcab3